### PR TITLE
Update app version to 4.3

### PR DESCRIPTION
**Pull request by rahmanshaber**

### CODE REVIEW

**Comment by hfiguiere** [comment]
build fail with 
```
FAILED: CMakeFiles/corefm.dir/settings.cpp.o 
/usr/bin/c++ -DQT_CORE_LIB -DQT_DBUS_LIB -DQT_GUI_LIB -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DVERSION_TEXT=\"4.3.0\" -I. -Icorefm_autogen/include -Iio -Imodels -Iui -Iviews -isystem /usr/include/QtCore -isystem /usr/lib/mkspecs/linux-g++ -isystem /usr/include/QtGui -isystem /usr/include/QtWidgets -isystem /usr/include/QtDBus -O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -O3 -DNDEBUG   -Wall -fPIC -std=gnu++17 -MD -MT CMakeFiles/corefm.dir/settings.cpp.o -MF CMakeFiles/corefm.dir/settings.cpp.o.d -o CMakeFiles/corefm.dir/settings.cpp.o -c settings.cpp
settings.cpp: In member function ‘int settings::autoUIMode() const’:
settings.cpp:158:55: error: ‘CPrime::FormFactor’ has not been declared
  158 |     if (CPrime::ThemeFunc::getFormFactor() == CPrime::FormFactor::Mobile) {
      |                                                       ^~~~~~~~~~
settings.cpp:160:62: error: ‘CPrime::FormFactor’ has not been declared
  160 |     } else if (CPrime::ThemeFunc::getFormFactor() == CPrime::FormFactor::Tablet &&  CPrime::ThemeFunc::getTouchMode() == true) {
      |                                                              ^~~~~~~~~~
settings.cpp:165:1: warning: control reaches end of non-void function [-Wreturn-type]
  165 | }
      | ^
```

**Comment by rahmanshaber** [comment]
@s96Abrar please merge this.

File: org.cubocore.CoreFM.appdata.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@
     <url type="contact">https://app.element.io/#/room/#cubocore:matrix.org</url>
     
     <releases>
-        <release version="4.2.0" date="2021-04-11"/>
+        <release version="4.3.0" date="2021-04-11"/>
```
Comment (s96Abrar) [inline org.cubocore.CoreFM.appdata.xml]: Update release date
```suggestion
        <release version="4.3.0" date="2021-12-25"/>
```

**Review by s96Abrar — CHANGES_REQUESTED** [review]
Update libcprime [tag](https://github.com/flathub/org.cubocore.CoreFM/blob/master/org.cubocore.CoreFM.yml#L33) version to 
```yml
tag: v4.3.0
```

**Review by s96Abrar — APPROVED** [review]

**Comment by s96Abrar** [comment]
bot, build org.cubocore.CoreFM

**Pull request merged**